.eslintrc

```diff
@@ -3,6 +3,7 @@
   "extends": ["next", "next/core-web-vitals", "prettier"],
   "rules": {
     "prettier/prettier": "error",
+    "endOfLine": "auto",
```

Comment:
Why do we need this?

Reply:
I was getting an error in vscode that was saying "Delete `cr`", and its because eslint (which is our linter for React) doesn't like a certain windows keycode. It's explained better in here: https://stackoverflow.com/questions/53516594/why-do-i-keep-getting-eslint-delete-cr-prettier-prettier.